SECURITY.md

```diff
@@ -1,18 +1,108 @@
-# Security Policy
+# Security Vulnerability Disclosure Policy
 
-## Reporting a Vulnerability
+## Introduction
 
-Please report (suspected) security vulnerabilities to security@intersectmbo.org. You will receive a
-response from us within 48 hours. If the issue is confirmed, we will release a patch as soon
-as possible.
+The Cardano open source project (xxx) is committed to ensuring the security of
+its software and the privacy of its users. We value the contributions
+of the security community in helping us identify and address
+vulnerabilities in our code. This Security Vulnerability Disclosure
+Policy outlines how security vulnerabilities should be reported and
+how we will respond to and remediate such reports.
 
-Please provide a clear and concise description of the vulnerability, including:
+## Security Vulnerability Handling Process
 
-* the affected version(s) of all packages included in ouroboros-network repository,
-* steps that can be followed to exercise the vulnerability,
-* any workarounds or mitigations
+### Reporting a Vulnerability
+
+If you discover a security vulnerability in xxxx, we encourage you to
+responsibly disclose it to us. To report a vulnerability, please use
+the [private reporting form on
+GitHub](https://github.com/input-output-hk/mithril/security/advisories/new)
```

Comment:
We certainly shouldn't use `input-output-hk/mithril`.  What I suggest below is also wrong since it's `404`:
```suggestion
GitHub](https://github.com/IntersectMBO/ouroboros-network/security/advisories/new)
```

Reply:
The link I posted is 404, and there's no way to create a new advisory in the [`ouroboros-network` repo](https://github.com/input-output-hk/mithril/security/advisories).  You'll need to configure it correctly.